api/types/system/info.go

```diff
@@ -160,3 +161,12 @@ type FirewallInfo struct {
 	// Info is a list of label/value pairs, containing information related to the firewall.
 	Info [][2]string `json:"Info,omitempty"`
 }
+
+// DeviceInfo represents a discoverable device from a device driver.
+type DeviceInfo struct {
+	// Source indicates the origin device driver.
+	Source string `json:"Source"`
+	// Name is the unique identifier for the device.
+	// Example: CDI FQDN like "vendor.com/gpu=0", or other driver-specific device ID
+	Name string `json:"Name"`
```

Comment:
Hmmm, perhaps `ID` would make more sense here to keep consistency with the actual `DeviceRequest` struct we use in our API.

https://github.com/moby/moby/blob/2154b9c6469f45aa689e32ee35e6d2b632ab0244/api/types/container/hostconfig.go#L263

@thaJeztah @tonistiigi  WDYT?

Reply:
ok with calling it smth else. It should have the value that I can use in `--device`. Doesn't `Source` make more sense as `Type`?